v10

WilliamDenniss opened the pull request:
Addressing changes from last call.

WilliamDenniss committed:
Fix typo, thanks Eric Fazendin for pointing it out.

WilliamDenniss committed:
Fix section reference as the referenced spec re-ordered the sections prior to publishing.

WilliamDenniss committed:
Resolve missing access_denied error issue.

Issue was raised by Andrew, text proposed by Brian.

WilliamDenniss committed:
Add linebreaks to avoid splitting the URN

WilliamDenniss committed:
Corrected the error message that is returned when the token expires.

Thanks to Brian Campbell for noticing this error!
Verified that this was the behavior that Google is applying.

selfissued commented on draft-ietf-oauth-device-flow.xml:
```diff
@@ -921,8 +933,8 @@
-        Justin Richer, Ken Wang, Steven E.&nbsp;Wright, Nat Sakimura, and
-        Torsten Lodderstedt.
+        Justin Richer, Ken Wang, Steven E.&nbsp;Wright, Nat Sakimura,
+        Torsten Lodderstedt, Eric Fazendin, and Andrew Sciberras.
```
Please alphabetize this list.

WilliamDenniss replied:
Done. I also added a credit for our chairs and ADs, and slightly clarified the credit for the earlier contributions.

selfissued reviewed (approved):
These changes look good.  I'd alphabetize the acknowledgements by last name before merging, as is standard to do.  Typically I put one name per line so as to minimize changes when more names are inserted into the list.  We'll be adding more during IETF and IESG review.

WilliamDenniss committed:
Added line-break to remove xml2rfc warning

WilliamDenniss committed:
Alphabetized (on last name) the acknowledgment list at Mike's request

WilliamDenniss committed:
Add a credit for the chairs and ADs, clarified the acknowledgement of prior contributors

– Seemed strange to thank those in the group through 2010, and then the same group later, so removed the reference to the date.

WilliamDenniss committed:
Bumped version, date, added Changelog

selfissued reviewed (approved):
Good to go!
